### PR TITLE
Run Command and Constants refactor 

### PR DESCRIPTION
**Pull request by MichaelKetchel**
## Type of change 
Refactor and code quality improvements

## Breaking change

*Does this Pull Request introduce a breaking change?* 
No.
## Proposed change

*Please describe the purpose of this change, the problem it solves, and why the change is necessary. Include code snippets or links to related documentation when applicable*
Refactoring things to use a unified, more hardened shell command runner pattern, along with moving some duplicated constants into a single place

## Testing

*Please explain how you tested this change manually, and if applicable, what new tests you added.*

Added unit tests for the new run_command functions along with some of the other util functions I added 

## Related Issues/PRs 
This PR aims to fix https://github.com/WLAN-Pi/wlanpi-core/issues/12

### CODE REVIEW

**Comment by MichaelKetchel** [comment]
@joshschmelzle This is ready for review i think. I've smoked tested what I can of the changes, they seem to be solid.

**Comment by joshschmelzle** [comment]
@MichaelKetchel can you check the failing Python style police status check. Looks like it's complaining about unused imports.

**Comment by MichaelKetchel** [comment]
i believe it's angry at my todo item i left to flag the blind exception handling: 
image

File: wlanpi_core/services/utils_service.py [hunk 1]
```diff
@@ -16,97 +17,103 @@
...
+    )
 
     # Browse Google.com
-    browse_google = run_command(
-        "timeout 2 curl -s -L www.google.com | grep 'google.com'"
+    browse_google_result = await browse_google_result_cr
+    output["results"]["Browse Google"] = (
+        "OK"
+        if (
+            browse_google_result.success and "google.com" in browse_google_result.stdout
```
[the remark this answers is not in the document]
Comment (MichaelKetchel) [reply wlanpi_core/services/utils_service.py:71]: Maybe I'm missing something, but this doesn't seem like a big deal?

**Pull request merged**